### DISCUSSION TITLE
Unit Test Tab on preview panel

### DISCUSSION THREAD

**Discussion by nukkamoto** [codesandbox/codesandbox-client#5089] [Q&A]
I have created a nodejs simple application with minimal functionality.
I have also created a file `xxxxxx.test.js` under source. however I cannot see a test tab to preview the test.

the only option is to run it on the terminal which is not as great as the test tab.

is there a way to force the tab to show or configure the project to allow the tests to be configured?

https://codesandbox.io/s/nodeplayground-gxuw6?file=/src/index.js

**Comment by garethx** [comment]
Jest integration only works in client sandboxes, see https://codesandbox.io/docs/tests

**Reply by nukkamoto** [reply]
I have seen that, is there any plan to do for nodejs server?
